### PR TITLE
Added two different hash that are '/reg' for student only and '/stude…

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
…nt-login' for student credential input fields.

### CODE REVIEW

File: src/components/StudentPage.js [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+      registeredStudentCredentialErrorMessage:false,
+    };
+
+    this._enableEditInfo = this.enableEditInfo.bind(this);
+    this._handleInputChange = this.handleInputChange.bind(this);
+  }
+
+  componentWillMount() {
+    const id = getParameterByName('id');
+    const secretCode = getParameterByName('secCode');
```
Comment (pepcus-pratik) [inline src/components/StudentPage.js]:
```suggestion
    const secretCode = getParameterByName('secretCode');
```

File: src/components/StudentPage.js [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+              {this.renderLoginField()}
+            </div>
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+SplashPage.propTypes = {
```
Comment (pepcus-pratik) [inline src/components/StudentPage.js]: Where are the **propTypes** and **defaultProps**??

File: src/components/StudentPage.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    if (this.state.isURLParams) {
+      return <Switch><Redirect to={'/studentCorrection'} /></Switch>
+    }
+    return (
+      <div className="landing-page-block">
+        <div className={'landingPageContainer'}>
+          <h2 className="student-heading">{yjsgHeader}</h2>
+        </div>
+        <div className="landing-page-wrapper">
+          <div className={'landingPageContent'}>
```
Comment (pepcus-pratik) [inline src/components/StudentPage.js]: Follow the class name convention

File: src/components/StudentPage.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  }
+
+
+  render() {
+    if (this.state.isURLParams) {
+      return <Switch><Redirect to={'/studentCorrection'} /></Switch>
+    }
+    return (
+      <div className="landing-page-block">
+        <div className={'landingPageContainer'}>
```
Comment (pepcus-pratik) [inline src/components/StudentPage.js]: Follow the class name convention

File: src/components/StudentPage.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+          <h2 className="student-heading">{yjsgHeader}</h2>
+        </div>
+        <div className="landing-page-wrapper">
+          <div className={'landingPageContent'}>
+            <div className={'yjsg-event-info'}>
+              <h5 className="primary-color">{eventDate}</h5>
+              <h5 className="header-text">{eventVenue}</h5>
+            </div>
+            <div className={'landingPageLogo'}>
```
Comment (pepcus-pratik) [inline src/components/StudentPage.js]: Follow the class name convention

File: src/components/StudentPage.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+              <img src={yjsgLogo} alt={'yjsg logo'} />
+            </div>
+            <div className={'landingPageButtonContainer'}>
```
Comment (pepcus-pratik) [inline src/components/StudentPage.js]: Follow the class name convention

Comment (BhagyashreeBhoyar) [reply src/components/StudentPage.js]: This will be done by Priyal.

File: src/components/StudentPage.js (hunk 5 above)
Comment (BhagyashreeBhoyar) [reply src/components/StudentPage.js]: This will be done by Priyal.

File: src/components/StudentPage.js (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/StudentPage.js]: This will be done by Priyal.

File: src/components/StudentPage.js (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/StudentPage.js]: This will be done by Priyal.

File: src/components/StudentPage.js (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/components/StudentPage.js]: Done.

File: src/components/StudentPage.js (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/components/StudentPage.js]: Done.

**Pull request merged**